app/services/create_member_for_api_members_controller.rb

```diff
@@ -27,6 +27,12 @@ def create
 
   private
 
+  def member_params
+    return @member_params if @member_params.present?
+    allowed = Member.new.attributes.keys.map(&:to_sym) + [:name]
+    @member_params = @params.select{ |k, v| allowed.include? k }
```

Comment:
Can't you just pass it to the service class separately? Do you really need to mass assign the locale to the member? You need to pass it as an argument to `@member.publish_subscription(@params[:locale])` in any case.